src/hotspot/share/utilities/concurrentHashTable.hpp

```diff
@@ -40,7 +40,7 @@
 class Thread;
 class Mutex;
 
-template <typename CONFIG, MEMFLAGS F>
+template <typename CONFIG, MemTag F>
```

Comment:
Parameter name should be updated throughout ConcurrentHashTable. Suggest mem_tag.